documentation/Migration/Update-0.109.md

```diff
@@ -146,8 +153,12 @@ var error = new ErrorInfo("error", "message")
 
 The following are changes that are not required, but may be useful to you.
 
-## CustomDataSourceBase
+## ICustomDataSource
+The `ICustomDataSource` is now marked obsolete and will be removed prior to a v1.0.0 release candidate.
+You should update all references to use the renamed `IProcessingSource`.
+
+## ProcessingSource (formerly CustomDataSourceBase)
 
 The method `SetApplicationEnvironmentCore` is now virtual, so you no longer have
```

Comment:
whoops, thought this method was on the interface. I will re-add the CustomDataSourceBase section with this paragraph.